src/pybricks/hubs.pyi

```diff
@@ -17,17 +17,20 @@ class EV3Brick:
 class MoveHub:
     battery: Battery
     light: ColorLight
+    system: System
 
 
 class CityHub:
     battery: Battery
     light: ColorLight
+    system: System
 
 
 class TechnicHub:
     def __init__(self, top_size: Axis, front_side: Axis): ...
     battery: Battery
     light: ColorLight
+    system: System
 
 
 class PrimeHub:
```

Comment:
It looks like we also need to add `EssentialHub` to this files.

Reply:
It has the following

```c
typedef struct _hubs_EssentialHub_obj_t {
    mp_obj_base_t base;
    mp_obj_t battery;
    mp_obj_t buttons;
    mp_obj_t charger;
    mp_obj_t imu;
    mp_obj_t light;
    mp_obj_t system;
} hubs_EssentialHub_obj_t;
```

Reply:
`imu` is also missing from all hubs that support it.

Reply:
How about a separate PR for these?

Also, `MoveHub` assigns `_SimpleAccelerometer()` to the `imu` property whereas `TechnicHub` and `PrimeHub` assign the `_IMU()` instance. A little weird that these two have such a different class names.

Reply:
Sure. The move hub literally just has an accelerometer whereas the other hubs have an IMU with accelerometer and gyro, thus the different classes.

Reply:
Thanks for explaining. Added `IMU` to supported Hubs. Will add the `EssentialHub` in a separate PR.